hebi_a-2084-01_moveit_config/config/A-2084-01.srdf

```diff
@@ -10,35 +10,37 @@
     <!--CHAINS: When a chain is specified, all the links along the chain (including endpoints) are included in the group. Additionally, all the joints that are parents to included links are also included. This means that joints along the chain and the parent joint of the base link are included in the group-->
     <!--SUBGROUPS: Groups can also be formed by referencing to already defined group names-->
     <group name="hebi_arm">
-        <joint name="HEBI/base/X5_4" />
-        <joint name="HEBI/shoulder/X5_9" />
-        <joint name="HEBI/elbow/X5_4" />
-        <joint name="HEBI/wrist1/X5_1" />
+        <joint name="Arm/J1_base/X5_4" />
+        <joint name="Arm/J2_shoulder/X5_9" />
+        <joint name="Arm/J3_elbow/X5_4" />
+        <joint name="Arm/J4_wrist1/X5_1" />
     </group>
     <!--GROUP STATES: Purpose: Define a named state for a particular group, in terms of joint values. This is useful to define states like 'folded arms'-->
-    <group_state name="home" group="hebi_arm">
-        <joint name="HEBI/base/X5_4" value="-1" />
-        <joint name="HEBI/elbow/X5_4" value="-2" />
-        <joint name="HEBI/shoulder/X5_9" value="0" />
-        <joint name="HEBI/wrist1/X5_1" value="0" />
-    </group_state>
```

Comment:
group_state got removed again (I'll just note this on each .srdf so we have it marked)